terraform/modules/github_actions_access/app_repo_policies.tf

```diff
@@ -0,0 +1,31 @@
+data "aws_iam_policy_document" "push_images" {
```

Comment:
Arguably could refactor to put this with the ecr definition, but it was already in this module so that seems a bit more effort than it's worth right now.